app/routes/requests--get.js

```diff
@@ -56,15 +59,27 @@ module.exports = [
         Request.find(filters).skip(skip).limit(req.limit).exec()
       ]).then(results => {
         var [count, rawRequests] = results;
-        return Promise.all(rawRequests.map(o => Task.find({requestId: o._id}, {status: true}).exec()))
+        return Promise.all(rawRequests.map(o => Task.find({requestId: o._id}, {status: true, geometry: true}).exec()))
           .then(allReqTasks => {
             // Add the task count to the requests.
             let requests = rawRequests.map((r, i) => {
+              let reqTasks = allReqTasks[i];
               // Convert from mongoose model to object.
               r = r.toObject();
+
+              if (req.query.footprint) {
+                let tasksWithGeo = reqTasks.filter(task => task.geometry !== null);
+                if (tasksWithGeo.length) {
+                  let fc = featureCollection(tasksWithGeo.map(task => polygon([task.geometry])));
+                  r.footprint = envelope(fc);
```

Comment:
I'm using `turf-bbox` elsewhere to get extent coordinates. Could `envelope` be combined with the `polygon` helper to let us remove a turf dependency, or is better to use the most explicit tool?